.github/workflows/ci-linux.yml

```diff
@@ -189,41 +171,11 @@ jobs:
           ["maximal"]
       docker_targets: "with-targets-optional"
       # [0-9a-o] excludes _, in particular package _develop
-      targets_optional: '$(echo $(export PATH=build/bin:$PATH && sage-package list :optional: --has-file "spkg-install.in|spkg-install|requirements.txt" --no-file "huge|has_nonfree_dependencies" | grep -v sagemath_doc | grep ^[0-9a-o]))'
-
-
-  optional-p-z:
-    if: ${{ success() || failure() }}
-    needs: [optional-0-o]
-    uses: ./.github/workflows/docker.yml
-    with:
-      incremental: true
-      free_disk_space: true
-      from_docker_repository: ghcr.io/${{ github.repository }}/
-      from_docker_target: "with-targets-pre"
-      tox_packages_factors: >-
-          ["maximal"]
-      docker_targets: "with-targets-optional"
-      # [0-9a-o] excludes _, in particular package _develop
-      targets_optional: '$(echo $(export PATH=build/bin:$PATH && sage-package list :optional: --has-file "spkg-install.in|spkg-install|requirements.txt" --no-file "huge|has_nonfree_dependencies" | grep -v sagemath_doc | grep ^[p-z]))'
-
-  experimental-0-o:
-    if: ${{ success() || failure() }}
-    needs: [optional-p-z]
-    uses: ./.github/workflows/docker.yml
-    with:
-      incremental: true
-      free_disk_space: true
-      from_docker_repository: ghcr.io/${{ github.repository }}/
-      from_docker_target: "with-targets-pre"
-      tox_packages_factors: >-
-          ["maximal"]
-      docker_targets: "with-targets-optional"
-      targets_optional: '$(echo $(export PATH=build/bin:$PATH && sage-package list :experimental: --has-file "spkg-install.in|spkg-install|requirements.txt" --no-file "huge|has_nonfree_dependencies" | grep -v sagemath_doc | grep ^[0-9a-o]))'
+      targets_optional: '$(echo $(export PATH=build/bin:$PATH && sage-package list :optional: --has-file "spkg-install.in|spkg-install|requirements.txt" --no-file "huge|has_nonfree_dependencies" | grep -v sagemath_doc))'
```

Comment:
may remove L173.

Reply:
Thanks - actually I had to fix something that that comment explained

Reply:
Ah really.